PocketKit/Sources/PocketKit/MyList/ArchivedItemsList/ArchivedItemsListViewModel.swift

```diff
@@ -390,8 +390,10 @@ extension ArchivedItemsListViewModel {
                 case .all:
                     return nil
                 case .tagged:
+                    filterTagOverflowAnalytics()
```

Comment:
🤔 Something about this seems incorrect. It's not exactly an overflow button (compared to the overflow button in an item itself). @cyndichin double checking here - what's correct for when "Tagged" is selected in the carousel?

Looking at the doc, though, it does appear that there's just a basic UI hierarchy of overflow button -> list screen (either my_list or archive). So… is this just an overloaded word?

Reply:
Yeah, you're right. i think I misunderstood the context of the analytics. ill fix that up later today.